angular-cli.json

```diff
@@ -17,7 +17,10 @@
       "styles": [
         "styles.css"
       ],
-      "scripts": [],
+      "scripts": [
```

Comment:
I feel like you know my pain. You can include scripts from node_modules here too. So might be able to do 
`scripts: ["../node_modules/av/src/aurora.js", "../node_modules/mp3/whatever`